src/util/gifRecorder.ts

```diff
@@ -0,0 +1,76 @@
+import { Recorder } from './recorder';
+import gifEncoder = require('gif-encoder-2-browser');
+import { createCanvas } from 'canvas';
+
+interface GifRecordingContext {
+  interval: NodeJS.Timer;
```

Comment:
```suggestion
  interval: ReturnType<typeof setInterval>;
```

Intellij infers a type from `NodeJS.Timer` which is incorrect when the GIF recorder is imported into a project targeting browser environments like a web view, but is at least consistent with the parameter type of `clearInterval`.
The proposed type declaration should work for all target environments.